src/components/shared/transactionPriority/transactionPriority.css

```diff
@@ -58,6 +58,10 @@
         padding: 5px var(--horizontal-padding-m);
         outline: none;
 
+        &:disabled {
+          background-color: var(--color-platinum);
+        }
+
```

Comment:
And this style

Reply:
How then we show visually that the priority option is disabled?

Reply:
Ah sorry. My bad